### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: data/hoverboard.config.json [hunk 1]
```diff
@@ -3,7 +3,10 @@
   "url": "https://hoverboard-v2-dev.firebaseapp.com/",
   "startDate": "2017-10-013T09:00:00+03:00",
   "endDate": "2017-10-14T19:00:00+03:00",
-  "analytics": "UA-43643469-8",
+  "analytics": {
+    "prod": "UA-43643469-1",
```
Comment (ozasadnyy) [inline data/hoverboard.config.json]: it's dfua code. Replace with `UA-43643469-8` and `UA-43643469-9` for dev

File: index.html [hunk 2]
```diff
@@ -74,6 +74,10 @@
 
   <script src="bower_components/firebase/firebase.js"></script>
   <script>
+
+    window.ENV = 'dev';
```
Comment (ozasadnyy) [inline index.html]: create `environment` variable in hoverboard.config

File: index.html [hunk 3]
```diff
@@ -74,6 +74,10 @@
...
+    window.PREFIX = '/';
```
Comment (ozasadnyy) [inline index.html]: same here

File: src/elements/hoverboard-analytics.html [hunk 4]
```diff
@@ -1,28 +1,81 @@
...
         } catch (e) {
+          // no-op
         }
-      };
+      };    
+
+      // HOTJAR
+      (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:407943,hjsv:5};
```
Comment (ozasadnyy) [inline src/elements/hoverboard-analytics.html]: remove hotjar from hoverboard. We will use it only on our website

File: src/elements/hoverboard-analytics.html [hunk 5]
```diff
@@ -1,28 +1,81 @@
...
+        a=o.getElementsByTagName('head')[0];
+        r=o.createElement('script');r.async=1;
+        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+        a.appendChild(r);
+      })(window,document,'//static.hotjar.com/c/hotjar-','.js?sv=');
+
+      // FACEBOOK PIXEL CODE
+      !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
```
Comment (ozasadnyy) [inline src/elements/hoverboard-analytics.html]: same here

File: index.html (hunk 2 above)
Comment (sophieH29) [inline index.html]: do you want to have like this in config?
"environment": {
    "dev": "dev",
    "prod": "prod"
  },

**Comment by sophieH29** [comment]
Not sure about offline now, will found out

File: src/elements/hoverboard-analytics.html [hunk 6]
```diff
@@ -1,28 +1,56 @@
...
+      // Load google analytis script
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
       (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
       m=s.getElementsByTagName('body')[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      // Load autotrack script
+      (function(s,o,g,a,m){a=s.createElement(o),m=s.getElementsByTagName('body')[0];
+      a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})
+      (document,'script','../../node_modules/autotrack/autotrack.js');
```
Comment (ozasadnyy) [inline src/elements/hoverboard-analytics.html]: add to `polymer.json` into `extraDependencies`
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/polymer.json#L24

Comment (sophieH29) [inline src/elements/hoverboard-analytics.html]: good point, thanks

**Pull request merged**